src/diffusers/models/autoencoder_kl.py

```diff
@@ -12,13 +12,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 from dataclasses import dataclass
-from typing import Optional, Tuple, Union
+from typing import Dict, Optional, Tuple, Union
 
 import torch
 import torch.nn as nn
 
 from ..configuration_utils import ConfigMixin, register_to_config
 from ..utils import BaseOutput, apply_forward_hook
+from .attention_processor import AttentionProcessor, AttnProcessor
```

Comment:
Should we maybe also implement this automatic discoverability of `AttnProcessor2_0`? 

https://github.com/huggingface/diffusers/blob/231bdf2e560a57976749b55a941bcf3f311df6fc/src/diffusers/models/attention_processor.py#L160

Reply:
I don't fully follow here. Note that by default the ATTN 2.0 is always chosen whenever someone uses the `Attention` class which is done for both the AutoEncoder and the Prior, see:
- https://github.com/huggingface/diffusers/blob/77f9137f103b04f137b6c487814ea58599ebc200/src/diffusers/models/attention_processor.py#L158
and:
- https://github.com/huggingface/diffusers/blob/231bdf2e560a57976749b55a941bcf3f311df6fc/src/diffusers/models/attention.py#L85